src/components/Login/LoginManager.jsx

```diff
@@ -1,18 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './LoginManager.css';
 
-class LoginManager extends React.Component {
+const LoginManager = () => {
+	const [email, setEmail] = useState('');
+	const [password, setPassword] = useState('');
 
-  constructor(props) {
-    super(props);
-  }
+	return (
+		<div className='container'>
+			<div className='card'>
+				<h1>3D Printing and Upload Queue</h1>
+				<h2>Sign In</h2>
+				<form>
+					<input
```

Comment:
@Patrickcpoole also include 'role' parameter on inputs for accessibility compliance